### PR TITLE
[4.0]Namespace com_contact

### PR DESCRIPTION
**Pull request by brianteeman**
Namespace of frontend components. Broken down into one PR for each component so that hopefully it can now be merged more easily and without the need for daily having to resolve merge conflicts.

### CODE REVIEW

File: components/com_contact/tmpl/contact/default_articles.php [hunk 1]
```diff
@@ -10,14 +10,16 @@
 defined('_JEXEC') or die;
 
 JLoader::register('ContentHelperRoute', JPATH_SITE . '/components/com_content/helpers/route.php');
+use Joomla\CMS\Router\Route;
```
Comment (Quy) [inline components/com_contact/tmpl/contact/default_articles.php]: Move above line 12.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b07cb0c5e634630b633f637567d5e85b6a225c43<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20911">issues.joomla.org/tracker/joomla-cms/20911</a>.</sub>

**Comment by brianteeman** [comment]
restarting drone so that hopefully the tests will not fail anymore

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**